src/__tests__/lib/power-to-energy.test.ts

```diff
@@ -0,0 +1,241 @@
+import {ERRORS} from '@grnsft/if-core/utils';
+
+import {ConvertPowerToEnergy} from '../../../if-run/builtins/power-to-energy';
+
+import {STRINGS} from '../../../if-run/config';
+
+const {InputValidationError, ConfigError} = ERRORS;
+const {MISSING_CONFIG} = STRINGS;
```

Comment:
These lines would not work as community-plugin. Did you test this on your forked community-plugin repo? GHA workflow reports build error.